src/main/java/core/basesyntax/classes/functional/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.classes.functional;
+
+import java.util.Random;
+

```

Comment:
Remove this unnecessary empty line to maintain a clean and organized code structure.